kt/settings.gradle.kts

```diff
@@ -1,6 +1,12 @@
 include("godot-library")
 
 pluginManagement {
+    repositories {
+        // jcenter()
+        mavenCentral() // use this instead
+        google()
+        gradlePluginPortal()
```

Comment:
why do we add `google` and `gradlePluginPortal` here ?

Reply:
When I was prototyping it I tried a few different options for downloads. One of them was a gradle plugin dedicated for that. Yes that needs to be removed.